services/movie.py

```diff
@@ -1,11 +1,13 @@
 from django.db.models import QuerySet
```

Comment:
There no no need a blank line between those two imports. They are retated to the same import group